README.md

```diff
@@ -24,21 +24,53 @@ ESLint is a tool for identifying and reporting on patterns found in ECMAScript/J
 * ESLint uses an AST to evaluate patterns in code.
 * ESLint is completely pluggable, every single rule is a plugin and you can add more at runtime.
 
-## Installation
+## Installation and Usage
 
-You can install ESLint using npm:
+There are two ways to install ESLint: globally and locally. 
 
-    npm install -g eslint
+### Local Installation and Usage
 
-## Usage
+If you want to include ESLint as part of your project's build system, we recommend installing it locally. You can do so using npm:
 
-If it's your first time using ESLint, you should set up a config file using `--init`:
+```
+$ npm install eslint
+```
+
+You should then setup a configuration file:
+
+```
+$ ./node_modules/.bin/eslint --init
+```
+
+After that, you can run ESLint on any file or directory like this:
+
+```
+$ ./node_modules/.bin/eslint yourfile.js
+```
+
+Any plugins or shareable configs that you use must also be installed locally to work with a locally-installed ESLint.
 
-    eslint --init
+### Global Installation and Usage
 
-After that, you can run ESLint on any JavaScript file:
+If you want to use ESLint with your code editor, or you want to make ESLint available to tools that run across all of your projects, we recommend installing ESLint globally. You can do so using npm:
```

Comment:
Code editors can detect local eslint. And it can be a source of problems to have different global/local versions, so I think that advise can be misleading and I would remove it.